templates/main/text_container.html

```diff
@@ -1,6 +1,11 @@
 {% extends "main::container.html" %}
 
 {% block article %}
+
+<script type="text/javascript" src="http://code.jquery.com/jquery-1.10.0.min.js"></script>
```

Comment:
JQuery is already loaded in `container.html` (https://github.com/Formulae-Litterae-Chartae/formulae-capitains-nemo/blob/12e91e97e4f87d6dc0d1c1f24e6639c509338aa3/templates/main/container.html#L80). I expect that that import makes this one unnecessary.

Reply:
I tested a bit locally and the problem is that you have the `<script>` element at the top of the page whereas jquery is loaded in `container.html` at the bottom of the page. That means that the page was trying to load and run your script before JQuery had been loaded. I was able to get it to work (along with the arrow on the right column flipping) by adding the following lines to the bottom of the `text_container.html` template:

```
{% block footer %}
<script type="text/javascript" src="{{url_for('InstanceNemo.static', filename='js/text_container.js')}}"></script>
{% endblock %}
```
Once you have done this, you can take your call to jquery at the top of the page and everything should work. Try it out and then we can discuss it when you are back from vacation.